jwst/lib/suffix.py

```diff
@@ -103,6 +103,8 @@
     'skymatch',
     'extract_2d',
     'cubebuildstep',
+    'residualfringstep',
+    'residual_fringe',
```

Comment:
The presence of this value may be causing the test to fail (but I'm not familiar with this suffix code, so maybe not).